src/main/java/frc/robot/Cal.java

```diff
@@ -73,8 +74,14 @@ public static final class Intake {
         DEPLOY_MIN_OUTPUT_VELOCITY_DEG_PER_SECOND = Cal.PLACEHOLDER_DOUBLE,
         DEPLOY_ALLOWED_CLOSED_LOOP_ERROR_DEG = Cal.PLACEHOLDER_DOUBLE;
 
-    /** Threshold for having achieved the desired intake position (in degrees) */
-    public static final double POSITION_THRESHOLD_DEGREES = 3.0;
+    /** Margin for having achieved the desired intake position (in degrees) */
+    public static final double POSITION_MARGIN_DEGREES = 3.0;
+
+    /** Positive and negative margins for softLimits for SparksMax */
```

Comment:
This comment doesn't add much beyond the variable name itself. You could say something like it sets the min and max positions that the intake will be allowed to reach